test-infra/camel-test-infra-ollama/src/test/java/org/apache/camel/test/infra/ollama/services/OllamaRemoteService.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.test.infra.ollama.services;
+
+import org.apache.camel.test.infra.ollama.commons.OllamaProperties;
+
+public class OllamaRemoteService implements OllamaService {
+
+    public OllamaRemoteService() {
+    }
+
+    public OllamaRemoteService(String baseURL, String model) {
+        System.setProperty(OllamaProperties.BASE_URL, baseURL);
+        System.setProperty(OllamaProperties.MODEL, model);
```

Comment:
This should be done on the `registerProperties`.

Reply:
@orpiske Or maybe for a RemoteService, this sis supposed to be set manually in the properties file?

Reply:
Typically we don't use properties files, so setting it using `System.setProperty` is usually OK. 

Obs.: If the remote service is too complicated to setup, properties files are OK, when set in the `registerProperties` method. We just never needed it.